pysparkling/sql/types.py

```diff
@@ -1840,31 +1822,47 @@ def _check_series_convert_timestamps_tz_local(s, timezone):
     byte=ByteType(),
     smallint=ShortType(),
     short=ShortType(),
-    int=LongType(),
-    integer=LongType(),
+    int=IntegerType(),
+    integer=IntegerType(),
```

Comment:
I am taking the opportunity to discuss this here: which one should we take?

Pyspark 2.4 converts 'integer' into IntegerType but Pyspark 3 converts 'integer' into LongType.

Both are ok with me, but I was wondering if the impact of this change was noticed

Reply:
This is too young a project to worry about backwards compatibility. If it does not hurt your own use case, I'd say go with PySpark 3 compatibility.